specification/azsadmin/resource-manager/containerregistry/Microsoft.ContainerRegistry.Admin/preview/2019-11-01/registries.json

```diff
@@ -0,0 +1,156 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-11-01",
+    "title": "ContainerRegistryManagementClient",
+    "description": "Admin Container Registry Management Client."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerRegistry.Admin/locations/{location}/registries": {
+      "get": {
+        "x-ms-examples": {
+          "Returns a list of container registries at the given location.": {
+            "$ref": "./examples/registries/List.json"
+          }
+        },
+        "tags": [
+          "ContainerRegistries"
+        ],
+        "description": "Returns a list of container registries present in all tenant location.",
+        "operationId": "ContainerRegistries_List",
+        "parameters": [
+          {
+            "$ref": "containerregistry.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The list of container registries has been returned.",
+            "schema": {
+              "$ref": "#/definitions/ContainerRegistriesList"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "containerregistry.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ContainerRegistry": {
+      "description": "Container registry properties.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "Container registry properties.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ContainerRegistryProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "containerregistry.json#/definitions/Resource"
```

Comment:
use appropriate common type

Reply:
we use "containerregistry.json for shared definitions and Resource is part of them

Reply:
See my note above, use one of the common types from here, rather than defining your own resource: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v2/types.json#L9-L84